.github/workflows/operate-e2e-tests.yml

```diff
@@ -17,6 +17,8 @@ on:
       - "parent/*"
       - "pom.xml"
   pull_request:
+    branches:
+      - "merge-optimize-in"
```

Comment:
I'll remove this before merging. I needed it to activate the workflon on pull request against the branch

Reply:
Do we have any mechanism to make sure we dont forget?

Reply:
Yes, I am the mechanism: I'll do it right after the  approval of this :P